core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/AutoscaleV4Controller.java

```diff
@@ -119,27 +120,31 @@ public AutoscaleStackV4Response getInternalAutoscaleClusterByName(String name, @
     }
 
     @Override
-    @CheckPermissionByAccount(action = AuthorizationResourceAction.POWERUSER_ONLY)
+    @DisableCheckPermissions
```

Comment:
I think these can be removed:
    @PreAuthorize("hasRole('AUTOSCALE')")

    @CheckPermissionByAccount(action = AuthorizationResourceAction.POWERUSER_ONLY)

Reply:
It was the original version before the change, I wouldn't risk it to introduce a new failure.